src/main/java/org/openmaptiles/layers/Landcover.java

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright (c) 2021, MapTiler.com & OpenMapTiles contributors.
+Copyright (c) 2023, MapTiler.com & OpenMapTiles contributors.
```

Comment:
I notice you've got 2023 in a few of these headers

Reply:
Ah, @TomPohys was asking about the same.

TL;DR: IANAL hence I tend to put there a year of "most recent substantial change". This particular file has such in 2023.

Longer: Looking through various IP advice and Open Source licensing FAQs, etc. there seems to be varying degree of advice, from "does not matter" to "really important", with various tactics about when and how to change that value. And since I'm not going to hire a layer (which is IMHO important if we truly need a legal advice about for example what "substantial change" means), then I outline my rationale:

1. Extreme position 1: Bump the copyright year every year even when no changes were done = IMHO totally bogus and IANAL but legally unenforceable.
2. Extreme position 1: Put there a year at the beginning and then never bump it, even if the file goes through substantial changes all the time = IMHO also wrong, but sort of "wronging myself" (e.g. possibly loosing protections which I'm entitled to, but hey, copyright protection is supposed to run even several years after my death, so ...).

Hence I opt to be "somewhere in between", e.g. bump the year when substantial changes occur.